docs/project-service/project-v1.api.yaml

```
@@ -0,0 +1,1047 @@
+          description: Project workflow update request accepted
+          schema:
+            $ref: '#/definitions/ProjectStatusResponse'
+        '202':
+          description: Project workflow update request accepted
+          schema:
+            $ref: '#/definitions/ProjectStatusResponse'
+        '400':
+          description: Invalid Input body
+          schema:
+            $ref: 'https://raw.githubusercontent.com/egovernments/egov-services/master/docs/common/contracts/v1-1-1.yml#/definitions/ErrorRes'
+# Remove the old /project/v1/workflow/update path definition if it exists. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)